src/Files.App/UserControls/StatusCenter/SpeedGraph.cs

```diff
@@ -96,17 +127,30 @@ private void Init()
 			width = size.X;
 			height = size.Y;
 
+			var rootClip = compositor.CreateRectangleClip();
+			rootClip.Top = 1.5f;
+			rootClip.Left = 1.5f;
+			rootClip.Bottom = height - 1.5f;
+			rootClip.Right = width - 2f;			// i fear i might be a victim of DPI scaling here
+			rootClip.TopLeftRadius = new(4f);
+			rootClip.TopRightRadius = new(4f);
+			rootClip.BottomLeftRadius = new(4f);
+			rootClip.BottomRightRadius = new(4f);
+			rootVisual.Clip = rootClip;
+
 			var accentColor = (App.Current.Resources["AccentFillColorDefaultBrush"] as SolidColorBrush)!.Color;
 
-			var backgroundBrush = compositor.CreateColorBrush(accentColor with { A = 0x15 });
+			backgroundBrush = compositor.CreateColorBrush(accentColor with { A = 0x0f });
 
 			var graphFillBrush = compositor.CreateLinearGradientBrush();
 			graphFillBrush.StartPoint = new(0.5f, 0f);
 			graphFillBrush.EndPoint = new(0.5f, 1f);
-			graphFillBrush.ColorStops.Add(compositor.CreateColorGradientStop(0f, accentColor with { A = 0x70 }));
-			graphFillBrush.ColorStops.Add(compositor.CreateColorGradientStop(1f, accentColor with { A = 0x06 }));
+			graphFillTop = compositor.CreateColorGradientStop(0f, accentColor with { A = 0x7f });
+			graphFillBottom = compositor.CreateColorGradientStop(1f, accentColor with { A = 0x0f });
```

Comment:
this seems to be the same value assigned to `backgroundBrush.Color`, might be better to make this value a variable and assign it to both